packages/block-library/src/block/reusable-block-welcome-guide.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+import { Guide } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+function WelcomeGuideImage( { nonAnimatedSrc, animatedSrc } ) {
```

Comment:
I have mixed feelings about this as well since we are not introducing a new concept here and `save` button/action used to be there before.. Also I've tried your PR a bit yesterday and today and both times I loaded a post the first time, it showed me the basic welcome guide. I didn't investigate why this happened though. Did this happen for anyone else?